jsdoc/index.html

```diff
@@ -0,0 +1,7 @@
+<!doctype html>
+<html>
+  <head>
+    <meta http-equiv="refresh" content="0; url=current">
```

Comment:
This redirects to `braintree-web-drop-in/docs/` to `braintree-web-drop-in/docs/current`. It's the same thing we do for `braintree-web` docs.